met-web/config-overrides.js

```diff
@@ -45,5 +59,128 @@ module.exports = function override(config) {
+                coreJs:{
+                    test: /[\\/]node_modules[\\/](core-js|core-js-pure|core-js-compat)[\\/]/,
+                    name: 'core-js',
+                    priority: 30,
+                    reuseExistingChunk: true,
+                },
+                redux: {
+                    test: /[\\/]node_modules[\\/](redux|react-redux|redux-thunk|@reduxjs)[\\/]/,
+                    name: 'redux',
+                    priority: 30,
+                    reuseExistingChunk: true,
+                },
+                dateFns: {
+                    test: /[\\/]node_modules[\\/]date-fns[\\/]/,
+                    name: 'date-fns',
+                    priority: 30,
+                    reuseExistingChunk: true,
+                },
+                formio: {
+                    test: /[\\/]node_modules[\\/](@formio|formiojs|met-formio|inputmask)[\\/]/,
+                    name: 'formio',
+                    priority: 20,
+                    reuseExistingChunk: true,
+                },
+                recharts: {
+                    test: /[\\/]node_modules[\\/](recharts|es-toolkit)[\\/]/,
+                    name: 'recharts',
+                    priority: 25,
+                    reuseExistingChunk: true,
+                },
+                jspdf: {
+                    test: /[\\/]node_modules[\\/](jspdf|html2canvas|html-to-image|canvg|stackblur-canvas)[\\/]/,
+                    name: 'jspdf',
+                    priority: 25,
+                    reuseExistingChunk: true,
+                },
+                draftjs: {
+                    test: /[\\/]node_modules[\\/](draft-js|draftjs-to-html|html-to-draftjs|draftjs-utils|react-draft-wysiwyg)[\\/]/,
+                    name: 'draftjs',
+                    priority: 25,
+                    reuseExistingChunk: true,
+                },
+                lodash: {
+                    test: /[\\/]node_modules[\\/](lodash|lodash-es)[\\/]/,
+                    name: 'lodash',
+                    priority: 25,
+                    reuseExistingChunk: true,
+                },
+                moment: {
+                    test: /[\\/]node_modules[\\/]moment[\\/]/,
+                    name: 'moment',
+                    priority: 15,
+                    reuseExistingChunk: true,
+                },
+                helloPangea: {
+                    test: /[\\/]node_modules[\\/](@hello-pangea)[\\/]/,
+                    name: 'hello-pangea',
+                    priority: 15,
+                    reuseExistingChunk: true,
+                },
+                // Remaining vendor libraries
+                vendor: {
+                    test: /[\\/]node_modules[\\/]/,
+                    name: 'vendors',
+                    priority: 10,
+                    reuseExistingChunk: true,
+                },
+                // Separate common code used across routes
+                common: {
+                    minChunks: 2,
+                    priority: 5,
```

Comment:
The priority system is a great tie-breaker, and I appreciate that you sorted the entries by priority.